configuration/configuration.json

```
@@ -33,7 +33,8 @@
                 "Message": "There has been a problem in creating your exchange set, so we are unable to fulfill your request at this time. Please contact UKHO Customer Services quoting correlation ID [jobid]"
             },
             "Response": {
-                "ExchangeSetExpiresIn": "7.00:00:00" // TimeSpan representing when the exchange set expires
+                "ExchangeSetExpiresIn": "7.00:00:00", // TimeSpan representing when the exchange set expires
+                "MaxExchangeSetSizeMB": 300 // Maximum total file size in MB for an exchange set
```

Please change in class as well
```suggestion
                "MaxExchangeSetSizeInMB": 300 // Maximum total file size in MB for an exchange set
```

---

Done